Sources/GRPC/CallHandlers/BidirectionalStreamingCallHandler.swift

```diff
@@ -25,7 +25,10 @@ import Logging
 ///   If the framework user wants to return a call error (e.g. in case of authentication failure),
 ///   they can fail the observer block future.
 /// - To close the call and send the status, complete `context.statusPromise`.
-public class BidirectionalStreamingCallHandler<RequestMessage: Message, ResponseMessage: Message>: BaseCallHandler<RequestMessage, ResponseMessage> {
+public class BidirectionalStreamingCallHandler<
```

Comment:
Do we consider these classes a public API or "an implementation detail of the codegen", i.e. could we issue a patch release that breaks compatibility _unless the user re-runs the codegen_?